lxc/storage_volume.go

```diff
@@ -1807,7 +1807,12 @@ func (c *cmdStorageVolumeShow) Command() *cobra.Command {
     Will show the properties of a custom volume called "data" in the "default" pool.
 
 lxc storage volume show default container/data
-    Will show the properties of the filesystem for a container called "data" in the "default" pool.`))
+    Provide the type of the storage volume if it is not custom.
```

Comment:
This would make more send moved up to above this specific example, and before the list of valid types that can be provided.

Reply:
Moved, thanks for the hint.